atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -224,17 +228,27 @@ def set_go_metadata(self, extra):
             extra['image']['go'] = go
 
     def set_operators_metadata(self, extra, worker_metadatas):
+        # update push plugin and uploaded manifests file independently as push plugin may fail
+        op_push_res = self.workflow.postbuild_results.get(PLUGIN_PUSH_OPERATOR_MANIFESTS_KEY)
+        if op_push_res:
+            extra.update({
+                "operator_manifests": {
```

Comment:
should this go in the uploaded manifests metadata though?

Reply:
IIRC we had discussion about this in design phase and agreed on this